includes/shortcodes.php

```diff
@@ -27,13 +27,14 @@ function tsml_next_meetings($arguments)
 	$meetings = array_slice($meetings, 0, $arguments['count']);
 	$rows = '';
 	foreach ($meetings as $meeting) {
-		$classes = tsml_to_css_classes($meeting['types']);
+		if (array_key_exists('types', $meeting))
+			$classes = tsml_to_css_classes($meeting['types']);
```

Comment:
looks good! two small suggestions:

when dealing with undefined key notices, we typically use `if (!empty(` as below because it won't throw a notice and will also tell us if there's a value there.

also i wonder if this could be combined with the two other if statements here like this:

```
$meeting_types = '';
if (!empty($meeting['types'])) {
  $classes = tsml_to_css_classes($meeting['types']);
  $meeting_types = tsml_format_types($meeting['types']);
}
```

then we could get rid of lines 37-40

Reply:
OK, Just committed that.